src/index.html

```diff
@@ -11,7 +11,212 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form method="post" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset
+      class="inputs-block"
+      >
+        <legend>
+      Personal information
+        </legend>
```

Comment:
```suggestion
        <legend>
          Personal information
        </legend>
```
Два отступа внутрь, а не наружу